packages/time-series/lib/commands/RANGE.ts

```diff
@@ -108,10 +108,10 @@ export default {
   IS_READ_ONLY: true,
   transformArguments: transformRangeArguments.bind(undefined, 'TS.RANGE'),
   transformReply: {
-    2(reply: UnwrapReply<ArrayReply<SampleRawReply[2]>>) {
-      return reply.map(sample => transformSampleReply['2'](sample));
+    2(reply: UnwrapReply<ArrayReply<SampleRawReply2>>) {
```

Comment:
`transformSamplesReply`?
also, do the unwrap inside of the generic transformer instead of in its args, so it'll be easier to use it..

Reply:
confusion mentioned above probably applies here as well.